docs/tutorials/nodes-tutorial.ipynb

```diff
@@ -0,0 +1,580 @@
+    "`smooth_node` asks nodes `in_node` and `slider_node` for their data.\n",
+    "`in_node` returns the raw data, while `slider_node` returns the integer value for the kernel size.\n",
+    "`smooth_node` then simply sends the inputs to the `gaussian_filter` function, and forwards the result to the figure."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b3a07b8f-7212-491f-a878-e854f23496e6",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-warning\">\n",
+    "\n",
+    "*Warning*\n",
+    "\n",
+    "The figure will not update when dragging the slider in the documentation pages.\n",
+    "This will only work inside a Jupyter notebook.\n",
+    "    \n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4375c234-a3c7-47a6-a48f-ee67837cbf82",
+   "metadata": {},
+   "source": [
+    "## Multiple views\n",
+    "\n",
+    "To go one step further,\n",
+    "we now wish to add a one-dimensional figure that will display the sum of the two-dimensional data along the vertical dimension.\n",
+    "On this figure, we would like to display both the original (unsmoothed) data, as well as the smoothed data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "686b3cdf-33b5-4e45-85b9-b2266211c0bc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "in_node = pp.input_node(da)\n",
+    "in_node.name = 'Input node'\n",
+    "\n",
+    "slider = ipw.IntSlider(min=1, max=20)\n",
+    "slider_node = pp.widget_node(slider)\n",
+    "slider_node.name = 'Slider node'\n",
+    "\n",
+    "smooth_node = pp.node(gaussian_filter)(in_node, sigma=slider_node)\n",
+    "smooth_node.name = 'Smoothing'\n",
+    "\n",
+    "fig2d = pp.figure2d(smooth_node)\n",
+    "\n",
+    "# Sum the raw data along the vertical dimension\n",
+    "sum_raw = pp.node(sc.sum, dim='y')(in_node)\n",
+    "sum_raw.name = 'Summing raw data'\n",
+    "# Sum the smoothed data along the vertical dimension\n",
+    "sum_smoothed = pp.node(sc.sum, dim='y')(smooth_node)\n",
+    "sum_smoothed.name = 'Summing smoothed data'\n",
+    "# Give two nodes to a figure to display both on the same axes\n",
+    "fig1d = pp.figure1d(sum_raw, sum_smoothed)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "36c9ea09-8701-465c-beec-f55d371192ee",
+   "metadata": {},
+   "source": [
+    "We check the graph again to make sure that the one-dimensional figure has two inputs,\n",
+    "and that both are performing a sum along the `y` dimension."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "81ae3b91-9622-48ca-bc90-f5cbce5ea426",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pp.show_graph(in_node)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d908a930-a1dc-4e7b-90f4-be8cb7e11d88",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ipw.VBox([slider, fig2d, fig1d])"
```

Comment:
Is it possible to start the slider at a different value than 1? As it stands, the lines look almost identical

Reply:
Changed to be at 10 when created.